web_custom_label/models/common.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2023 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from typing import List, Mapping
+
+
+def set_custom_labels_on_fields(labels: List[dict], fields: Mapping[str, dict]):
```

Comment:
I'd suggest to move all those functions into some abstract model, or move them into the base model with some prefix like `_web_custom_label` - your current approach makes it more difficult than necessary to change the behavior of this module (ORM and type hints mix just fine)